src/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: [], // initial state is an empty array
+  reducers: {
+    // reducers are functions that determine changes to the state
+    checkCategory: () => 'Under construction',
+  },
+});
+
+export const { checkCategory } = categoriesSlice.actions;
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Define%20an%20action%20type%20for%20checking%20the%20status.), Please define an action type for checking the status using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).